master/internal/api_tasks_intg_test.go

```diff
@@ -25,9 +25,7 @@ func errTaskNotFound(id string) error {
 	return status.Errorf(codes.NotFound, "task not found: %s", id)
 }
 
-var (
-	authzCommand *mocks.NSCAuthZ
-)
+var authzCommand *mocks.NSCAuthZ
```

Comment:
Correct me if this isn't just cosmetic, but it kinda feels like this sort of cleanup oughtta be in a separate PR.  Same with the `var id =` to `id :=` changes. :)

Reply:
agreed. this should be a noOp here anyway the target branch [already has it](https://github.com/determined-ai/determined/commit/7d9f25538e40ebb0ec193298f11656a4da0c8aaf) not sure why githug ui wouldn't show an updated view of it

Reply:
rebased and this is an issue no longer!